index.html

```diff
@@ -31,17 +34,147 @@
+          <il><a href="index.html"><img src="img/Tagruby2.png"></a></il>
+        </ul> 
+        <nav class="profesional-art-nav-2">
+          <a href="index.html"><img src="img/seeproject.png"></a>
+        </nav>
+      </div>
+      <div class="profesional-art-container-3">
+        <h2 class="profesional-art-h2-3">Profesional Art<br>Printing Data</h2>
+        <p class="profesional-art-p-3">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+        <ul class="profesional-art-ul-3">
+          <il><a href="index.html"><img src="img/Taghtml2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagbootstrap2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagruby2.png"></a></il>
+        </ul> 
+        <nav class="profesional-art-nav-3">
+          <a href="index.html"><img src="img/seeproject.png"></a>
+        </nav>
+      </div>
+      <div class="profesional-art-container-4">
+        <h2 class="profesional-art-h2-4">Profesional Art<br>Printing Data</h2>
+        <p class="profesional-art-p-4">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+        <ul class="profesional-art-ul-4">
+          <il><a href="index.html"><img src="img/Taghtml2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagbootstrap2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagruby2.png"></a></il>
+        </ul> 
+        <nav class="profesional-art-nav-4">
+          <a href="index.html"><img src="img/seeproject.png"></a>
+        </nav>
+      </div>
+      <div class="profesional-art-container-5">
+        <h2 class="profesional-art-h2-5">Profesional Art<br>Printing Data</h2>
+        <p class="profesional-art-p-5">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+        <ul class="profesional-art-ul-5">
+          <il><a href="index.html"><img src="img/Taghtml2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagbootstrap2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagruby2.png"></a></il>
+        </ul> 
+        <nav class="profesional-art-nav-5">
+          <a href="index.html"><img src="img/seeproject.png"></a>
         </nav>
       </div>
     </section>
+    <section class="main-about-me-section">
+      <div class="about-me-container">
+        <h1 class="about-me-h1">About me</h1>
+        <p class="about-me-p">Hello I’m a software developer! I can help<br>you build a product , feature or website<br>Look through some of my work and<br>experience! If you like what you see and<br>have a project you need coded, don’t<br>hesitate to contact me.</p>
+        <a href="index.html"><img src="img/getmyresume.png" alt="" class="getmyresume"></a>
```

Comment:
- [ ] Please I suggest wrapping the `a` tag with the `button` tag since it is a button to improve user experience. 😄 
- [ ] Please I suggest implementing hover effects for the button per project requirements. 
- [ ] This is a screenshot of the hover effects interactions. 

![Screenshot (288)](https://user-images.githubusercontent.com/78104544/236593758-63f40a2c-6d93-4c51-8793-9f22704e2654.png)

- [ ] This is a screenshot of the project requirements. 

![Screenshot (290)](https://user-images.githubusercontent.com/78104544/236593898-6e0262d5-58a5-48fa-a86c-7a68b1b545ea.jpg)